Engines/Wine/Verbs/gdiplus/script.js

```diff
@@ -18,19 +19,42 @@ class GDIPlus {
         const wizard = this.wine.wizard();
         const prefixDirectory = this.wine.prefixDirectory();
         const system32directory = this.wine.system32directory();
+        const architecture = this.wine.architecture();
 
         const setupFile = new Resource()
             .wizard(wizard)
-            .url(
-                "http://download.microsoft.com/download/a/b/c/abc45517-97a0-4cee-a362-1957be2f24e1/WindowsXP-KB975337-x86-ENU.exe"
-            )
-            .checksum("b9a84bc3de92863bba1f5eb1d598446567fbc646")
-            .name("WindowsXP-KB975337-x86-ENU.exe")
+            .url("https://download.microsoft.com/download/0/A/F/0AFB5316-3062-494A-AB78-7FB0D4461357/windows6.1-KB976932-X86.exe")
+            .checksum("c3516bc5c9e69fee6d9ac4f981f5b95977a8a2fa")
+            .name("windows6.1-KB976932-X86.exe")
             .get();
+      
+        new CabExtract()
+            .archive(setupFile)
+	    .wizard(wizard)
+            .to(`${prefixDirectory}/drive_c/gdiplus/`)
+            .extract(["-L", "-F", "x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll"]);
+		
+        cp(`${prefixDirectory}/drive_c/gdiplus/x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll`, system32directory);
+	
+        if (architecture == "amd64") {
+	    const system64directory = this.wine.system64directory();
+            const setupFile64 = new Resource()
```

Comment:
A newline in between `system64directory` and `setupFile64` would make the code easier readable in my opinion